src/phases/query/Multiplanner.yml

```diff
@@ -147,3 +147,94 @@ Hide14IndexesTemplate:
         - {OperationName: RunCommand, OperationCommand: {collMod: *coll2, index: {hidden: true, name: index14}}}
         - {OperationName: RunCommand, OperationCommand: {collMod: *coll2, index: {hidden: true, name: index15}}}
         - {OperationName: RunCommand, OperationCommand: {collMod: *coll2, index: {hidden: true, name: index16}}}
+
+fraction: &fraction {^NumExpr: {withExpression: "resultCount / docCount", andValues: {
+                      resultCount: &resultCount {^Parameter: {Name: "resultCount", Default: {unused: "Invalid result count."}}},
+                      docCount: {^Parameter: {Name: "docCount", Default: {unused: "Invalid docCount."}}}}}}
+lowBound: &lowBound {^NumExpr: {withExpression: "random * (1 - fraction)", andValues: {
+                      random: 0.5,
```

Comment:
For now, I have `random` set manually to 0.5. This is because `^NumExpr` is preprocessed, making it incompatible with `^RandomDouble` which is a generator. Furthermore, each use of a "variable" `^RandomDouble` is a literal substitution that re-generates a new random double, making it challenging to reference the same random number more than once. 

Can anyone think of a workaround to this problem? For now, the workload is functional but doesn't average out to the expected N.

Reply:
Off the top of my head: Would `^FixedGeneratedValue` be useful? It calls the generator once and fixes the value it gets.

Reply:
Unfortunately, this also doesn't work because the preprocessor attempts to parse the `^NumExpr` before the generation, and it errors out upon receiving "{^FixedGeneratedValue: {fromGenerator: {^RandomDouble: {min: 0.0, max: 1.0}}}}" rather than a numerical value.

```
genny.tasks.preprocess.ParseException: Invalid values for 'andValues' in '^NumExpr', only numerical values are allowed.
Node eval: {'random': {'^FixedGeneratedValue': {'fromGenerator': {'^RandomDouble': {'distribution': 'uniform', 'min': 0.0, 'max': 1.0}}}}, 'fraction': 0.0001}
```

Reply:
Resolved as discussed in standup.